containers/src/Kokkos_ScatterView.hpp

```diff
@@ -954,9 +988,24 @@ class ScatterView<DataType, Kokkos::LayoutRight, DeviceType, Op,
 
   template <typename... Dims>
   ScatterView(std::string const& name, Dims... dims)
-      : internal_view(view_alloc(WithoutInitializing, name),
+      : ScatterView(view_alloc(execution_space(), name), dims...) {}
+
+  template <typename... P, typename... Dims>
```

Comment:
I would appreciate it if we could add a comment here saying something like 
"This overload allows specifying an execution space instance to be used by passing, e.g., 
Kokkos::view_alloc(exec_space, "label") as first argument.

Reply:
Done, thanks. There were three of them.